sefaria/helper/crm/salesforce.py

```diff
@@ -110,7 +110,9 @@ def mark_as_spam_in_crm(self, uid):
         except:
             # log
             return False
-        return res
+
```

Comment:
Do we want to add the logging here?

Reply:
no. I actually like it better in user.py, but then let's delete that comment as well. and now that line 28 &30 are actually running things make more sense.

Reply:
Ok, removed all the log comments! Thanks shanee!